docs/guides/developer/docs/installation/source-linux.md

```diff
@@ -0,0 +1,120 @@
+Developer installation guide
+===========================
+
+These instructions outline how to install an all in one Opencast system on Linux.
+
+
+Get Opencast source:
+
+You can get the Opencast source code by either downloading a tarball of the source code or by cloning the Git
+repository. The latter option is more flexible, it is easier to upgrade and in general preferred for developers. The
+prior option, the tarball download, needs less tools and you do not have to download nearly as much as with Git.
+
+Using the tarball:
+
+Select the tarball for the version you want to install
+from the [GitHub releases section](https://github.com/opencast/opencast/releases).
+
+    # Download desired tarball
+    curl -OL https://github.com/opencast/opencast/archive/[...].tar.gz
+    tar xf [...].tar.gz
+    cd opencast--[...]
+
+Cloning the Git repository:
+
+    git clone https://github.com/opencast/opencast.git
+    cd opencast
+    git tag   <-  List all available versions
```

Comment:
Devs usually don't need tags and we shouldn't point to them by default. Usually, `develop` should be fine. Maybe point shortly at release branches.